stylesheets/stylesheet.css

```diff
@@ -878,4 +882,3 @@ footer a:hover {
 
 /* Mobile Portrait Size to Mobile Landscape Size (devices and browsers) */
 @media only screen and (max-width: 479px) { }
```

Comment:
The line was there before your commit.
And every line of code is ours not of any particular person.
If there is a bug and you can fixit you should.